src/pages/Order.js

```diff
@@ -1,12 +1,13 @@
 import OrderItem from "../components/OrderItem";
 import "./Order.css";
 
-export default function Order({ items, order, deleteButtonHandler }) {
-  const orderItems = order.map((orderItem) => {
-    const menuItem = items.filter((item) => {
-      return item.id === orderItem.id;
-    });
-    orderItem = { ...orderItem, ...menuItem[0] };
+export default function Order({ items, deleteButtonHandler }) {
+  let total = 0;
+  let orderItems = items.filter((item) => {
+    return item.quantity > 0;
+  });
+  orderItems = orderItems.map((orderItem) => {
+    total = total + orderItem.price * orderItem.quantity;
```

Comment:
Hey Debs, you could use `.toFixed(2)` here, so the total is showing 4.90€ instead of 4.9€  :)))
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed

Reply:
nice! I was looking for that! Thanks!